src/models.py

```diff
@@ -311,13 +311,13 @@ def build_embeddings(args, vocab, tasks, pretrained_embs=None):
     token_embedders = {}
     # Word embeddings
     n_token_vocab = vocab.get_vocab_size("tokens")
-    if args.word_embs != "none":
-        if args.word_embs in ["glove", "fastText"] and pretrained_embs is not None:
+    if not args.input_module.startswith("bert") and args.input_module not in ["gpt", "ELMo"]:
```

Comment:
This doesn't need to be a conditional here. We can just replace the else case at the end with an assert that checks that the argument is valid.